rascal-vscode-extension/README-lib.md

```diff
@@ -0,0 +1,9 @@
+# Rascal Metaprogramming Language - VSCode DSL Runtime library
+
+This package is intended writing your own VS Code extension while using
+[Rascal's VS Code LSP abstraction](https://github.com/usethesource/rascal-language-servers).
+
+This package extracts the [`ParameterizedLanguageServer`](https://github.com/usethesource/rascal-language-servers/blob/main/rascal-vscode-extension/src/lsp/ParameterizedLanguageServer.ts) class from the [Rascal VS Code extension](https://marketplace.visualstudio.com/items?itemName=usethesource.rascalmpl).
```

Comment:
don't understand the language. "This package extracts ... " What is it exactly that is encapsulated? The Readme also defines two LSP servers earlier and gives them names. Perhaps that would help if we use the same terms here for the same concepts?

Reply:
I've improved the wording in this file.